sdks/python/apache_beam/runners/dataflow/dataflow_exercise_streaming_metrics_pipeline_test.py

```diff
@@ -115,7 +115,9 @@ def run_pipeline(self):
     return dataflow_exercise_streaming_metrics_pipeline.run(argv)
 
   # Need not run streaming test in batch mode.
-  @attr('IT', 'ValidatesRunner', 'sickbay-batch')
+  @pytest.mark.no_sickbay_batch
+  @pytest.mark.it_validatesrunner
+  @pytest.mark.it_postcommit
```

Comment:
This is the only test that runs in both ValidatesRunner and post-commit. I think that's a bug but let's leave it for this PR.